### PR TITLE
Add Visual Element extensions

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `Behaviors VisualElement extension` <https://github.com/CommunityToolkit/Maui.Markup/issues/29>

### CODE REVIEW

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/visual-element-extensions.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Behaviors
+
+The `Behaviors` method adds the supplied behaviors to the `Behaviors` collection on the current `VisualElement`.
+
+The follow example will create an `Entry` and add a [`MaxLengthReachedBehavior`](../../behaviors/maximum-length-reached-behavior.md) to it.
+
+```csharp
+new Entry().Behaviors(new MaxLengthReachedBehavior());
+```
```
Comment (jfversluis) [inline docs/maui/markup/extensions/visual-element-extensions.md:99]: We don't have the "Examples" and "API" headings here as we do in the `view-in-absolute-layout-extensions` and all other pages?

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/visual-element-extensions.md (hunk 1 above)
Comment (bijington) [reply docs/maui/markup/extensions/visual-element-extensions.md:99]: We don't have explicit examples in the Markup sample app like we do with the Maui sample app. Do you think it is enough to point them to the sample application where some bits are used throughout?

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/visual-element-extensions.md (hunk 1 above)
Comment (bijington) [reply docs/maui/markup/extensions/visual-element-extensions.md]: I've updated to point to the entire application for now

Comment (jfversluis) [reply docs/maui/markup/extensions/visual-element-extensions.md]: Oh yeah if we don't have anything useful I guess we can skip it :D just stood out to me that it wasn't included here.

**Pull request merged**